### PR TITLE
Mark local test as pass or fail

### PR DESCRIPTION
**Pull request by emalysz**

### CODE REVIEW

**Comment by emalysz** [comment]
@francisf Please review

File: specs/local_test.js [hunk 1]
```diff
@@ -27,9 +27,21 @@ describe('BrowserStack Local Testing for ' + caps.browserName, function() {
...
-        assert(source.match(/Up and running/i) != null);
-        done();
+      driver.getPageSource().then(async function(source) {
+        try {
+          assert(source.match(/Up and running/i) != null);
+          //marking the test as Passed if source matches
+          await driver.executeScript(
+            'browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"passed","reason": "Source matches!"}}'
+          );
+          done();
```
Comment (francisf) [inline specs/local_test.js]: done needs to be called in finally

Comment (francisf) [reply specs/local_test.js]: + for selenium-4 branch

**Comment by emalysz** [comment]
I figured the changes could either be rebased or a separate PR could handle the Selenium 4 changes. The second PR is here for the selenium 4 branch: https://github.com/browserstack/mocha-browserstack/pull/8

**Review by francisf — APPROVED** [review]

**Pull request merged**